docs/produce_config_page.py

```diff
@@ -0,0 +1,64 @@
+from pathlib import Path
+import time
+import yaml
+import git
+from tabulate import tabulate
+
+
+def load_config(filename):
+    with open(filename, 'r') as f:
+        return yaml.safe_load(f)
+
+
+def add_prod_table(production_dir, prod_file='productions.rst'):
+    """
+
+    Parameters
+    ----------
+    production_dir: pathlib.Path or str
+        path to the production configs directory
+    prod_file: str
+        RST prod file to update
+    """
+
+    prod_txt = '\n'
+
+    commit_times = []
+    prod_list = []
+    for prod_dir in Path(production_dir).iterdir():
+        commit = list(git.Repo(root_dir).iter_commits(paths=prod_dir, max_count=1))[0]
+
+        yml_list = [f for f in prod_dir.iterdir() if f.name.endswith('.yml')]
+        if yml_list:
+            try:
+               conf = load_config(yml_list[0])
+               prod_id = conf['prod_id']
+            except:
+                print(f"Could not load prod id for {prod_dir.name}")
+                prod_id = ''
+        else:
+            print(f"No yml file in {prod_dir}")
+
+        prod_list.append([time.ctime(commit.committed_date),
+                          f"`{prod_dir.name} <{lstmcpipe_repo_prod_config_url+prod_dir.name}>`_",
+                          prod_id]
+                         )
+        commit_times.append(commit.committed_date)
+
+    sorted_lists = sorted(zip(commit_times, prod_list))
+    prod_list = [prod for _,prod in sorted_lists]
+
+    prod_txt += tabulate(prod_list, ['Request date', 'Directory name', 'Prod ID'], tablefmt='rst')
+
+    with open(prod_file, 'a') as f:
+        f.write(prod_txt)
+
+
+
+if __name__ == '__main__':
+    root_dir = Path(__file__).parent.joinpath('..').resolve()
+    prod_dir = root_dir.joinpath('production_configs')
+
+    lstmcpipe_repo_prod_config_url = 'https://github.com/cta-observatory/lstmcpipe/tree/master/production_configs/'
```

Comment:
`lstmcpipe_repo_prod_config_url` is also a var inside the `add_prod_table` function, but you don't pass it as argument. 
Is this really working ?

Reply:
it's working but bad practice, I will change